app/controllers/pages_controller.rb

```diff
@@ -12,7 +12,9 @@ def home
   	}
 
     @hosts = sort_by_field(@hosts, params[:sort] || 'user.full_name')
-
+    if params[:reverse_ordering].to_i == 0
+      @hosts = @hosts.reverse
```

Comment:
if i applied it in the sort_by_field, it would have to be applied to each header, with a conditional on each.